tests/settings.py

```diff
@@ -127,7 +127,9 @@
 JWT_ISSUER = 'issuer'
 JWT_ID_ATTRIBUTE = 'username'
 
-JWT_PRIVATE_KEY_RSA_ISSUER = """
+#JWT_ENC_ALGORITHM = 'HS256' #'RS256'] #would still be default
+
+JWT_SECRET_ISSUER = """
```

Comment:
I would keep it `JWT_PRIVATE_KEY_ISSUER` to make the name similar to `JWT_PUBLIC_KEY_ISSUER`. When we talk about keys, the usual terms are "private" and "public"

Reply:
Before I push these changes should we provide the possibility for HMAC-users to omit the `JWT_PUBLIC_KEY_` and therefore default back to `JWT_PRIVATE_KEY_`.

This would mean at the current state, that if an other suite than HMAC is set and the `JWT_PUBLIC_KEY_` is omitted this would cause a misleading Exception `jwt.exceptions.InvalidKeyError`, but not `django.core.exceptions.ImproperlyConfigured: Missing setting JWT_PUBLIC_KEY_ISSUER`

Reply:
@Menda: ping

Reply:
Sorry, I was on vacation. By now I don't know anyone who uses HMAC, so we can omit it by now and just create an issue for that.

Reply:
And as I told you before, let's just keep `PRIVATE_KEY` instead `SECRET`. Then I will approve.

Reply:
> And as I told you before, let's just keep `PRIVATE_KEY` instead `SECRET`. Then I will approve.

I reverted to that naming scheme already just waited to push the changes, so I wouldn't have to commit 7 times again.